examples/flask/airbrake_middleware.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
split code that is specific to the Flask app and airbrake integration into 2 different files